### PR TITLE
Bump version and amend notes to include latest PRs

### PR DESCRIPTION
**Pull request by miekg**
Signed-off-by: Miek Gieben <miek@miek.nl>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]
LGTM, nice.

File: notes/coredns-1.8.4.md [hunk 1]
```diff
@@ -32,15 +36,18 @@ Yury Tsarev.
 
 ## Noteworthy Changes
 
+
```
Comment (SuperQ) [inline notes/coredns-1.8.4.md]: Extra newline?

File: notes/coredns-1.8.4.md [hunk 2]
```diff
@@ -32,15 +36,18 @@ Yury Tsarev.
...
 * core: fix reverse zones expansion (https://github.com/coredns/coredns/pull/4538)
+* plugins: fix Normalize (https://github.com/coredns/coredns/pull/4621)
 * plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
 * plugin/bind: Exclude interface or ip address  (https://github.com/coredns/coredns/pull/4543)
 * plugin/dnssec: heck for two days of remaining validity (https://github.com/coredns/coredns/pull/4606)
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]:
```suggestion
* plugin/dnssec: Check for two days of remaining validity (https://github.com/coredns/coredns/pull/4606)
```

File: notes/coredns-1.8.4.md [hunk 3]
```diff
@@ -32,15 +36,18 @@ Yury Tsarev.
...
 * plugin/dnssec: interface type correction for `periodicClean` sig validity check (https://github.com/coredns/coredns/pull/4608)
 * plugin/dnssec: use entire RRset as key input (https://github.com/coredns/coredns/pull/4537)
+* plugin/etcd: Fix etcd to use v3.5.0-alpha.0 (https://github.com/coredns/coredns/pull/4628)
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]: ~v3.5.0-beta.1~ v3.5.0-beta.2 just got released.

File: notes/coredns-1.8.4.md [hunk 4]
```diff
@@ -3,13 +3,17 @@ title = "CoreDNS-1.8.4 Release"
...
 author = "coredns"
 +++
 
 The CoreDNS team has released
 [CoreDNS-1.8.4](https://github.com/coredns/coredns/releases/tag/v1.8.4). This release includes a
-bunch of bugfixes and a few enhancements, and a new (small) plugin called *minimal*.
+bunch of bugfixes and a few enhancements mostly in the *dnssec* and *kubernetes* plugins, and a new
+(small) plugin called *minimal*.
+
+It also include a fix when using the "reverse zone syntax", i.e. 10.0.0.0/15, now return the proper
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]:
```suggestion
It also include a fix when using the "reverse zone cidr syntax", e.g. 10.0.0.0/15, now return the proper
```

**Review by chrisohaver — CHANGES_REQUESTED** [review]
LGTM + some small fixes, and also etcd is releasing 3.5 this week, they just dropped 3.5.0beta. Probably should at least bump to latest beta ...

**Comment by chrisohaver** [comment]
> ... etcd is releasing 3.5 this week ...

Clarification - they _started the release process this week_, code freeze yesterday. Reading the 3.5.0 beta CHANGELOG, looks like the final release is slated for June, which is a ways off.

File: notes/coredns-1.8.4.md (hunk 3 above)
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]:
```suggestion
* plugin/etcd: Bump etcd to v3.5.0-beta.3 (https://github.com/coredns/coredns/pull/4638)
```

**Comment by chrisohaver** [comment]
Given that #4621 will require updates in almost all external plugins, should we bump the minor version?  I.e. to v1.9.0 instead of v1.8.4.

File: notes/coredns-1.8.4.md (hunk 4 above)
Comment (chrisohaver) [inline notes/coredns-1.8.4.md]: Should highlight required changes to existing external plugins introduce with the plugin level reverse zone changes.

File: notes/coredns-1.8.4.md [hunk 5]
```diff
@@ -32,15 +36,18 @@ Yury Tsarev.
 
 ## Noteworthy Changes
 
+
 * core: fix reverse zones expansion (https://github.com/coredns/coredns/pull/4538)
+* plugins: fix Normalize (https://github.com/coredns/coredns/pull/4621)
 * plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md:42]:
```suggestion
* reverse zone: make Normalize return proper reverse zones (https://github.com/coredns/coredns/pull/4621)
* plugin/bind: Bind by interface name (https://github.com/coredns/coredns/pull/4522)
```

File: notes/coredns-1.8.4.md [hunk 6]
```diff
@@ -3,13 +3,17 @@ title = "CoreDNS-1.8.4 Release"
...
 +++
 
 The CoreDNS team has released
 [CoreDNS-1.8.4](https://github.com/coredns/coredns/releases/tag/v1.8.4). This release includes a
-bunch of bugfixes and a few enhancements, and a new (small) plugin called *minimal*.
+bunch of bugfixes and a few enhancements mostly in the *dnssec* and *kubernetes* plugins, and a new
+(small) plugin called *minimal*.
+
+It also include a fix when using the "reverse zone syntax", i.e. 10.0.0.0/15, now return the proper
+set of reverse zones.
```
Comment (chrisohaver) [inline notes/coredns-1.8.4.md:17]:
```suggestion
**NOTE:** This release deprecates `Host.Normalize()` and `Host.MustNormalize()`. For external plugins that use these functions, see https://github.com/coredns/coredns/pull/4621 for examples of how to update.
```

**Review by chrisohaver — CHANGES_REQUESTED** [review]

**Pull request merged**